cg/store/api/find_business_data.py

```diff
@@ -396,29 +412,29 @@ def get_metrics_entry_by_flow_cell_name_sample_internal_id_and_lane(
             lane=lane,
         ).first()
 
-    def get_flow_cell_by_name(self, flow_cell_name: str) -> Flowcell:
+    def filter_flow_cell_by_name(self, flow_cell_name: str) -> Flowcell:
```

Comment:
I though we use the patterns: when returning a query we use `filter` and when returning objects we use `get`

so `get_flow_cell_by_name` would be the name to use here.

Reply:
You are right! A S&R got out of hand - reverted 👍